src/Yaapii.Zip/Encrypted.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using Yaapii.Atoms;
+using Yaapii.Atoms.Error;
+
+namespace Yaapii.Zip
+{
+    /// <summary>
+    /// A zip archive, encrypted.
+    /// </summary>
+    public sealed class Encrypted : IInput
```

Comment:
This is one of two classes which deal with encryption.

1. ZipEncrypted -> Encrypts one file
2. Encrypted -> Encrypts every file in a given ZipArchive

Also there are two classes for decrypting:

1. Decrypted -> Decrypts every file in a given zip archive and returns the whole archive
2. ZipDecrypted -> Decrypts one file in a zip archive

I find this counter intuitive. Maybe its just the naming:
Encrypted -> EncryptedArchive (takes the whole zip and replaces every entry with an encrypted entry)
ZipEncrypted -> remove (I do not think we need archives with some files encrypted, and some not)

Decrypted -> remove (I do not think we need to revert encryption of a whole archive)
ZipDecrypted -> stays as it is